webcomponents/tool/src/main/frontend/packages/sakai-rubrics/test/sakai-rubrics.test.js

```diff
@@ -227,20 +226,20 @@ describe("sakai-rubrics tests", () => {
       </sakai-rubric-criterion-edit>
     `);
 
-    await waitUntil(() => el.querySelector("button.edit"), "edit button does not exist");
+    await waitUntil(() => el.querySelector("button.edit-criterion-button"), "edit button does not exist");
     expect(el.querySelector(`#edit-criterion-${data.criterion1.id}`)).to.exist;
     expect(el.querySelector("sakai-editor")).to.exist;
-    const button = el.querySelector("button.edit");
-    expect(button.getAttribute("title")).to.equal(`Edit Criterion ${data.criterion1.title}`);
+    const button = el.querySelector("button.edit-criterion-button");
+    //expect(button.getAttribute("title")).to.equal(`Edit Criterion ${data.criterion1.title}`);
+    expect(button.getAttribute("title")).to.equal(`Edit Criterion`);
```

Comment:
Is ok `Edit Criterion` or should be i18n?

Reply:
It could be.

Reply:
It now is i18n